### ISSUE TITLE
Fix/review `Exit with code 1 due to network error: ContentNotFoundError` - seems to be a PNG file type issue..?

### ISSUE THREAD

**Issue by ldexterldesign** [tagatac/bagoup#70]
👋

Hope you're OK!

Thanks for useful software!

**Describe the bug**

As title

Seems to export everything as expected until the (PNG related) errors appear and app exists (i.e. stops exporting)

```
bagoup --contacts-path ~/Desktop/foo.vcf --export-path messages-export-pdf --pdf --copy-attachments
2024/11/15 06:38:48 WARN: no valid text or attributedBody for message 40556
2024/11/15 06:38:49 WARN: chat file "messages-export-pdf/+44...561/iMessage;-;+44...561.pdf" - convert HEIC file "/messages-export-pdf/+44...561/attachments/Screenshot 2024-11-12 at 01.52.01.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/Screenshot 2024-11-12 at 01.52.01.jpeg": open HEIC file "/messages-export-pdf/+44...561/attachments/Screenshot 2024-11-12 at 01.52.01.heic": open /messages-export-pdf/+44...561/attachments/Screenshot 2024-11-12 at 01.52.01.heic: no such file or directory
2024/11/15 06:38:52 WARN: copy "/Users/ldexterldesign/Library/Messages/Attachments/a7/07/at_0_65704A8C-04F4-401F-B38A-AD53417063F6/IMG_0466.MOV" to "messages-export-pdf/+44...940/attachments" - "IMG_0466.MOV" already exists; using "IMG_0466-1.MOV" instead
2024/11/15 06:38:52 WARN: copy "/Users/ldexterldesign/Library/Messages/Attachments/ef/15/F88EE3B2-8810-493F-8C0E-14D025B424E8/IMG_0466.mov" to "messages-export-pdf/+44...940/attachments" - "IMG_0466.mov" already exists; using "IMG_0466-2.mov" instead
2024/11/15 06:38:52 WARN: copy "/Users/ldexterldesign/Library/Messages/Attachments/a8/08/at_1_65704A8C-04F4-401F-B38A-AD53417063F6/IMG_0466.MOV.3gp" to "messages-export-pdf/+44...940/attachments" - "IMG_0466.MOV.3gp" already exists; using "IMG_0466.MOV-1.3gp" instead
2024/11/15 06:38:52 WARN: copy "/Users/ldexterldesign/Library/Messages/Attachments/a7/07/at_0_65704A8C-04F4-401F-B38A-AD53417063F6/IMG_0466.MOV" to "messages-export-pdf/+44...940/attachments" - "IMG_0466.MOV" already exists; using "IMG_0466-3.MOV" instead
2024/11/15 06:38:59 WARN: no valid text or attributedBody for message 5854
2024/11/15 06:38:59 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - message 5854 - application/octet-stream attachment "F22557F6-B0DA-4B42-87F1-D8ADBA8314B2.pluginPayloadAttachment" (ID 508) - attachment does not exist locally
2024/11/15 06:38:59 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - message 5854 - application/octet-stream attachment "E146B5B4-AEAA-4C2F-9665-7C4C21E941CC.pluginPayloadAttachment" (ID 509) - attachment does not exist locally
2024/11/15 06:38:59 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/IMG_2676.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/IMG_2676.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/IMG_2676.heic": open /messages-export-pdf/+44...946/attachments/IMG_2676.heic: no such file or directory
2024/11/15 06:38:59 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/IMG_7737.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/IMG_7737.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/IMG_7737.heic": open /messages-export-pdf/+44...946/attachments/IMG_7737.heic: no such file or directory
2024/11/15 06:38:59 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/IMG_2714.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/IMG_2714.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/IMG_2714.heic": open /messages-export-pdf/+44...946/attachments/IMG_2714.heic: no such file or directory
2024/11/15 06:38:59 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/IMG_2716.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/IMG_2716.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/IMG_2716.heic": open /messages-export-pdf/+44...946/attachments/IMG_2716.heic: no such file or directory
2024/11/15 06:39:00 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/IMG_7985.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/IMG_7985.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/IMG_7985.heic": open /messages-export-pdf/+44...946/attachments/IMG_7985.heic: no such file or directory
2024/11/15 06:39:00 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/IMG_0229.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/IMG_0229.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/IMG_0229.heic": open /messages-export-pdf/+44...946/attachments/IMG_0229.heic: no such file or directory
2024/11/15 06:39:00 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/IMG_0169.HEIC" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/IMG_0169.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/IMG_0169.HEIC": open /messages-export-pdf/+44...946/attachments/IMG_0169.HEIC: no such file or directory
2024/11/15 06:39:00 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/73367982124__8CA106D2-C7A7-4CB0-ABE3-DEF55C86AC43.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/73367982124__8CA106D2-C7A7-4CB0-ABE3-DEF55C86AC43.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/73367982124__8CA106D2-C7A7-4CB0-ABE3-DEF55C86AC43.heic": open /messages-export-pdf/+44...946/attachments/73367982124__8CA106D2-C7A7-4CB0-ABE3-DEF55C86AC43.heic: no such file or directory
2024/11/15 06:39:00 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/IMG_0327.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/IMG_0327.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/IMG_0327.heic": open /messages-export-pdf/+44...946/attachments/IMG_0327.heic: no such file or directory
2024/11/15 06:39:00 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/image0.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/image0.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/image0.heic": open /messages-export-pdf/+44...946/attachments/image0.heic: no such file or directory
2024/11/15 06:39:00 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/View recent photos.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/View recent photos.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/View recent photos.heic": open /messages-export-pdf/+44...946/attachments/View recent photos.heic: no such file or directory
2024/11/15 06:39:00 WARN: no valid text or attributedBody for message 26059
2024/11/15 06:39:01 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/IMG_0686.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/IMG_0686.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/IMG_0686.heic": open /messages-export-pdf/+44...946/attachments/IMG_0686.heic: no such file or directory
2024/11/15 06:39:01 WARN: chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" - convert HEIC file "/messages-export-pdf/+44...946/attachments/IMG_8422.heic" to JPG: convert HEIC file to JPG file "/var/folders/zx/78tk00zs3nb1qbnwc6w6xx040000gn/T/bagoup778177203/IMG_8422.jpeg": open HEIC file "/messages-export-pdf/+44...946/attachments/IMG_8422.heic": open /messages-export-pdf/+44...946/attachments/IMG_8422.heic: no such file or directory
2024/11/15 06:39:01 WARN: no valid text or attributedBody for message 27140
2024/11/15 06:39:02 BAGOUP RESULTS:
bagoup version: 2.4.10 Darwin/arm64
Invocation: bagoup --contacts-path /Users/ldexterldesign/Desktop/foo.vcf --export-path messages-export-pdf --pdf --copy-attachments
Export folder: "messages-export-pdf"
Export files written: 13
Chats exported: 18
Valid messages exported: 614
Invalid messages exported (see warnings above): 1
Attachments copied: 146
	image/png: 8
	application/octet-stream: 106
	image/jpeg: 6
	image/heic: 14
	video/quicktime: 10
	video/3gpp: 2
Attachments referenced or embedded: 148
	video/quicktime: 10
	video/3gpp: 2
	image/png: 8
	application/octet-stream: 108
	image/jpeg: 6
	image/heic: 14
Attachments embedded: 14
	image/jpeg: 6
	image/png: 8
Attachments missing (see warnings above): 2
HEIC conversions completed: 0
HEIC conversions failed (see warnings above): 14
Time elapsed: 21.2318465s
2024/11/15 06:39:02 ERROR: export chats: flush chat file "messages-export-pdf/+44...946/iMessage;-;+44...946;;;SMS;-;+44...946.pdf" to disk: Loading pages (1/6)
Error: Failed to load file:///messages-export-pdf/+44...946/attachments/Screenshot 2023-10-15 at 23.23.31.png, with network status code 203 and http status code 0 - Error opening /messages-export-pdf/+44...946/attachments/Screenshot 2023-10-15 at 23.23.31.png: No such file or directory
Error: Failed to load file:///messages-export-pdf/+44...946/attachments/Screenshot 2023-10-25 at 00.24.31.png, with network status code 203 and http status code 0 - Error opening /messages-export-pdf/+44...946/attachments/Screenshot 2023-10-25 at 00.24.31.png: No such file or directory
Error: Failed to load file:///messages-export-pdf/+44...946/attachments/Screenshot 2023-11-08 at 02.04.47.png, with network status code 203 and http status code 0 - Error opening /messages-export-pdf/+44...946/attachments/Screenshot 2023-11-08 at 02.04.47.png: No such file or directory
Error: Failed to load file:///messages-export-pdf/+44...946/attachments/Screenshot 2023-11-08 at 02.44.34.png, with network status code 203 and http status code 0 - Error opening /messages-export-pdf/+44...946/attachments/Screenshot 2023-11-08 at 02.44.34.png: No such file or directory
Error: Failed to load file:///messages-export-pdf/+44...946/attachments/Screenshot 2023-11-08 at 02.54.51.png, with network status code 203 and http status code 0 - Error opening /messages-export-pdf/+44...946/attachments/Screenshot 2023-11-08 at 02.54.51.png: No such file or directory
Warning: Failed to load file:///messages-export-pdf/+44...946/attachments/IMG_0259.PNG (ignore)
Warning: Failed to load file:///messages-export-pdf/+44...946/attachments/IMG_0260.PNG (ignore)
Error: Failed to load file:///messages-export-pdf/+44...946/attachments/Screenshot 2024-05-20 at 23.56.09.png, with network status code 203 and http status code 0 - Error opening /messages-export-pdf/+44...946/attachments/Screenshot 2024-05-20 at 23.56.09.png: No such file or directory
Counting pages (2/6)
Resolving links (4/6)
Loading headers and footers (5/6)
Printing pages (6/6)
Done
Exit with code 1 due to network error: ContentNotFoundError

exit status 1
```

**To Reproduce**

Please see CLI output above

**Expected behavior**

Complete export and 

**Observed behavior**
A clear and concise description of what you observed to be different from what
you expected.

Incomplete export

**Screenshots**

```
tree /Users/ldexterldesign/Desktop/foo/messages-export-pdf
/Users/ldexterldesign/Desktop/foo/messages-export-pdf
├── +32498244839
│   ├── SMS;-;+32498244839.pdf
│   └── attachments
├── +35799715020
│   ├── SMS;-;+35799715020.pdf
│   └── attachments
├── +44...133
│   ├── SMS;-;+44...133.pdf
│   └── attachments
├── +44...496
│   ├── SMS;-;+44...496.pdf
│   └── attachments
├── +44...261
│   ├── attachments
│   └── iMessage;-;+44...261;;;SMS;-;+44...261.pdf
├── +44...881
│   ├── attachments
│   └── iMessage;-;+44...881.pdf
├── +44...561
│   ├── attachments
│   │   ├── 17CAC1CE-4FF5-4B61-AEDE-52DF4FA5E28D.pluginPayloadAttachment
│   │   ├── 20C69CCB-72DA-4AE9-8F6A-F524A187C32A.pluginPayloadAttachment
│   │   ├── 2F24BE72-2579-4A2E-B315-DCB47E80206A.pluginPayloadAttachment
│   │   ├── 369AF579-3957-4A19-B28D-14D18C99D52D.pluginPayloadAttachment
│   │   ├── 488572ED-5391-4652-B610-FCC7776636C6.pluginPayloadAttachment
│   │   ├── 4EBD5DD6-219B-4AF2-9A7A-F32ABF8D95D0.pluginPayloadAttachment
│   │   ├── 5E662419-6322-4EDE-8D86-5E926ECAF0A6.pluginPayloadAttachment
│   │   ├── 66A00488-82DD-4C1E-B5AC-5C11FD06F2A0.pluginPayloadAttachment
│   │   ├── 6800658D-D983-4A64-BAB8-CA2CD92B38AF.pluginPayloadAttachment
│   │   ├── 7A90BA0D-F882-4D93-B8C5-6ABD66C17606.pluginPayloadAttachment
│   │   ├── 7EB72E13-93D0-4E03-9417-3211424B7D23.pluginPayloadAttachment
│   │   ├── 81BE2817-54CB-4769-8E1A-D90E9C3D2AED.pluginPayloadAttachment
│   │   ├── 8D9E0D0C-2D8E-46AE-BCA8-7E288C1F93FF.pluginPayloadAttachment
│   │   ├── C29F21BB-37CA-4F3A-AC79-E99AEC09D1BE.pluginPayloadAttachment
│   │   ├── C595F9F5-37DD-40A8-A501-13B599893D37.pluginPayloadAttachment
│   │   ├── C9CC076C-ACC5-4030-A5B4-631DC346F40B.pluginPayloadAttachment
│   │   ├── CB43D1F6-607E-4A06-862C-3EDD76C823AD.pluginPayloadAttachment
│   │   ├── FF3A7046-0184-4FA8-9412-59A4A8E2E2EC.pluginPayloadAttachment
│   │   ├── IMG_1046.jpeg
│   │   ├── IMG_1968.jpeg
│   │   └── Screenshot 2024-11-12 at 01.52.01.heic
│   └── iMessage;-;+44...561.pdf
├── +44...253
│   ├── SMS;-;+44...253.pdf
│   └── attachments
│       ├── 7e8baa07-b14d-46d8-93b9-373cae86de291.jpg
│       ├── CE7E15FD-3E31-439E-B2A6-ED019D1DE563.pluginPayloadAttachment
│       ├── F8955830-F2A2-4276-989A-A4ACA92D7932.pluginPayloadAttachment
│       └── image0000001.jpg
├── +44...940
│   ├── attachments
│   │   ├── IMG_0466-1.MOV
│   │   ├── IMG_0466-2.mov
│   │   ├── IMG_0466-3.MOV
│   │   ├── IMG_0466.MOV-1.3gp
│   │   ├── IMG_0466.MOV.3gp
│   │   ├── IMG_0466.mov
│   │   ├── IMG_1492.mov
│   │   └── IMG_2425.JPG
│   └── iMessage;-;+44...940;;;SMS;-;+44...940.pdf
├── +44...830
│   ├── attachments
│   │   ├── 161A5B55-E4FD-4513-A0A1-E0DC8775239E.pluginPayloadAttachment
│   │   ├── 2EFF55CC-0C2E-4461-9344-2036D935C762.pluginPayloadAttachment
│   │   ├── 64BD926E-6694-4081-9127-AFA87B921F37.pluginPayloadAttachment
│   │   ├── 67E0ED13-79F4-4264-A348-C3F42B2997BD.pluginPayloadAttachment
│   │   ├── 961A9D28-9CA5-47F9-90F6-00EA28707A49.pluginPayloadAttachment
│   │   ├── 9CC5641A-68E0-4DF8-BE55-D4054C0FFE13.pluginPayloadAttachment
│   │   ├── AE276DE1-0A66-40DE-BA59-BC2C6F0E53AE.pluginPayloadAttachment
│   │   ├── EE79C065-247B-43E6-AD98-B5BF55F04122.pluginPayloadAttachment
│   │   └── IMG_1555.jpeg
│   └── iMessage;-;+44...830.pdf
├── +44...190
│   ├── SMS;-;+44...190.pdf
│   └── attachments
├── +44...403
│   ├── attachments
│   └── iMessage;-;+44...403;;;SMS;-;+44...403.pdf
├── +44...140
│   ├── SMS;-;+44...140.pdf
│   └── attachments
└── +44...946
    ├── attachments
    │   ├── 05F6CEC3-8CF5-49AD-80EE-B4FABB04C3B4.pluginPayloadAttachment
    │   ├── 09DEE097-F9E0-45E4-87A5-894AD3D7E823.pluginPayloadAttachment
    │   ├── 0F43EF9B-415E-418D-B36D-87590B37FC8F.pluginPayloadAttachment
    │   ├── 10245AD6-9908-4FAD-B37D-3664CA312F18.pluginPayloadAttachment
    │   ├── 136F5520-A81F-40C6-ABA6-AA7542CC8CB8.pluginPayloadAttachment
    │   ├── 1473D40D-447E-4EB7-8F4E-3A3E94A9D9C9.pluginPayloadAttachment
    │   ├── 1772FB27-B2FC-4D15-BC01-36ECD1047A57.pluginPayloadAttachment
    │   ├── 18B47E80-D592-4A32-A5A5-FE2DB0AF5B2C.pluginPayloadAttachment
    │   ├── 1F21F637-7826-49E5-9386-E591563D1CC1.pluginPayloadAttachment
    │   ├── 1FCAF6AB-EE60-47BB-90A5-2930D8B21E93.pluginPayloadAttachment
    │   ├── 25A57E92-7539-42A1-AE6C-0891DD42A59F.pluginPayloadAttachment
    │   ├── 2674593B-ECBF-412E-A034-640281DD6698.pluginPayloadAttachment
    │   ├── 274971C0-9D0A-4D8F-AE3A-9EE52C3F91EA.pluginPayloadAttachment
    │   ├── 29047CF6-B4B6-4243-BE96-E7FA22BAE07D.pluginPayloadAttachment
    │   ├── 2AB06604-BE33-446A-AA7A-EC90BA6BD15C.pluginPayloadAttachment
    │   ├── 2C279CF6-5D6F-473D-8B51-74BBA4EABA50.pluginPayloadAttachment
    │   ├── 2CEAEAD4-881D-460B-9857-9F7B5D56EBBA.pluginPayloadAttachment
    │   ├── 2CFA1363-ED73-4B5D-8B46-430EA1EF2D96.pluginPayloadAttachment
    │   ├── 2DB92A28-D1E3-4F86-8A48-4A45AD9D8999.pluginPayloadAttachment
    │   ├── 2E7018A7-9860-4B32-ABA7-E09E63C31A7F.pluginPayloadAttachment
    │   ├── 30999C26-D0FB-4394-A332-3F4BCF02C8D4.pluginPayloadAttachment
    │   ├── 31215DA6-71B0-463C-B639-C664C5FB01A7.pluginPayloadAttachment
    │   ├── 334592C4-A8D6-4478-A438-8DC4B7573A53.pluginPayloadAttachment
    │   ├── 3781C65D-60E1-4AAA-B84C-D93F2DF23C71.pluginPayloadAttachment
    │   ├── 396022A9-6EA5-476C-9211-FD5CB026EA00.pluginPayloadAttachment
    │   ├── 4370EE63-5705-447F-86B9-4F127443FA09.pluginPayloadAttachment
    │   ├── 4758333C-D7F0-430A-A06A-3DB472A3A8CF.pluginPayloadAttachment
    │   ├── 4C94DBB6-2186-4CCC-9567-27C20F6596D1.pluginPayloadAttachment
    │   ├── 5261FA89-A40B-4D19-9CD5-DB1C69315B34.pluginPayloadAttachment
    │   ├── 5945C563-EB01-4BA9-92B5-B7EA70F3A0FE.pluginPayloadAttachment
    │   ├── 59EAA616-3C84-4E33-9AAA-46A2141BAB9C.pluginPayloadAttachment
    │   ├── 5A2085DC-1647-4CD2-A0AD-89816AADA1D3.pluginPayloadAttachment
    │   ├── 5E5C8442-6D17-4F86-93DC-C9ABCE9021AF.pluginPayloadAttachment
    │   ├── 6013D4B6-58BD-49F1-9295-97D1A0D8EB83.pluginPayloadAttachment
    │   ├── 676FAC47-372E-49FF-91BC-640FA53D510B.pluginPayloadAttachment
    │   ├── 6D90E02A-76EB-401A-90AE-EBD22131BA21.pluginPayloadAttachment
    │   ├── 708B2725-ABDD-4AEE-AA12-EE621174124F.pluginPayloadAttachment
    │   ├── 71F853CA-293F-4E65-966E-DDDF9AB08005.pluginPayloadAttachment
    │   ├── 72333554-FE2A-43DA-9A2A-C4D4CB34C873.pluginPayloadAttachment
    │   ├── 73367982124__8CA106D2-C7A7-4CB0-ABE3-DEF55C86AC43.heic
    │   ├── 76A09E27-4B25-4EB7-AF55-65A3FBB09FBF.pluginPayloadAttachment
    │   ├── 793FE989-36E8-4389-9762-9BCD0802E047.pluginPayloadAttachment
    │   ├── 79700CA5-E7B6-4E2C-A352-99648AAE7580.pluginPayloadAttachment
    │   ├── 810412D7-5C51-44E6-B995-B5BD7C4DC023.pluginPayloadAttachment
    │   ├── 815565FA-ED37-4209-A08B-CEA39A0F0136.pluginPayloadAttachment
    │   ├── 8441C06F-72A6-45D7-AB63-F3AE8AFA7646.pluginPayloadAttachment
    │   ├── 855C2F72-C9B8-4F89-9065-CC4F6E1CA559.pluginPayloadAttachment
    │   ├── 93B8C50D-7250-4F61-9555-63BA8CEA4304.pluginPayloadAttachment
    │   ├── 9A082446-E48E-45D7-BE21-4F409C69B19A.pluginPayloadAttachment
    │   ├── A359645C-7A30-422F-80E3-85C57ED5F2E4.pluginPayloadAttachment
    │   ├── A9FC2723-0757-4B60-BEF6-9AD98BCAAA51.pluginPayloadAttachment
    │   ├── AA93C2B1-85A3-433A-939D-731F6B99C158.pluginPayloadAttachment
    │   ├── AB603A24-7C52-451B-B855-119F1079E566.pluginPayloadAttachment
    │   ├── ADBF5C21-B1B9-4804-B43C-5B52DCC955FD.pluginPayloadAttachment
    │   ├── Audio Message.caf
    │   ├── B024892B-AACD-4E65-BFE0-63F564530ABC.pluginPayloadAttachment
    │   ├── B2E9F20F-0B12-4E28-9A9B-82F4803EE106.pluginPayloadAttachment
    │   ├── B367FA1A-2B4F-4EB3-80C3-1EB57F0275E8.pluginPayloadAttachment
    │   ├── B39A5F28-DE89-4B35-BF1D-4D456D29CA72.pluginPayloadAttachment
    │   ├── B53D2728-87D9-42E1-992D-CF992BA08077.pluginPayloadAttachment
    │   ├── BC59ED50-2C93-4841-8569-75D44510772C.pluginPayloadAttachment
    │   ├── BCF00030-447D-4F1E-9BFD-2FCA55273ADD.pluginPayloadAttachment
    │   ├── C1858567-41EE-42B3-A671-24CBE795FED0.pluginPayloadAttachment
    │   ├── CCFF5814-045A-4AD8-8E00-B81BF26539A6.pluginPayloadAttachment
    │   ├── CFBC467A-2530-4491-A99B-A628F60F9E11.pluginPayloadAttachment
    │   ├── D24BFF20-2E75-4244-813F-E9C008BF0D3C.pluginPayloadAttachment
    │   ├── DB913C98-AEF2-44CD-9984-5A6328A4FEE5.pluginPayloadAttachment
    │   ├── E74CD1CC-2FF4-43FC-91F0-A8758890FA7E.pluginPayloadAttachment
    │   ├── E8266450-850B-4AA9-9941-51BE9B1D03C0.pluginPayloadAttachment
    │   ├── EDE1A37F-D7B4-40A0-9EE2-20E285F822A8.pluginPayloadAttachment
    │   ├── EEB28F4E-4660-4A95-B0F9-9F8433F6C28A.pluginPayloadAttachment
    │   ├── EmojiMovie737299966.mov
    │   ├── F0361C89-54A7-4A92-A745-829C5370AB67.pluginPayloadAttachment
    │   ├── F16A23CF-7EA9-4672-ACCE-C815D0EDE5A0.pluginPayloadAttachment
    │   ├── F1E930FD-2BA9-440E-89E0-4DDCC04B523B.pluginPayloadAttachment
    │   ├── F21574C3-1667-40E3-8D80-D0E1DE1C8BA8.pluginPayloadAttachment
    │   ├── F90EDA39-D4EB-4A31-A550-E18AF3DE785F.pluginPayloadAttachment
    │   ├── F9503207-108A-455E-9065-2B43732CB1AE.pluginPayloadAttachment
    │   ├── FBD3AB7E-90C1-44CB-B834-98CDDDD37CF4.pluginPayloadAttachment
    │   ├── FF7F594E-A711-4BE4-AB6F-D797D1A7CFB4.pluginPayloadAttachment
    │   ├── IMG_0006.mov
    │   ├── IMG_0169.HEIC
    │   ├── IMG_0229.heic
    │   ├── IMG_0259.PNG
    │   ├── IMG_0260.PNG
    │   ├── IMG_0327.heic
    │   ├── IMG_0372.MOV
    │   ├── IMG_0686.heic
    │   ├── IMG_1321.mov
    │   ├── IMG_2676.heic
    │   ├── IMG_2714.heic
    │   ├── IMG_2716.heic
    │   ├── IMG_2717.MOV
    │   ├── IMG_7737.heic
    │   ├── IMG_7985.heic
    │   ├── IMG_8422.heic
    │   ├── Screenshot 2023-10-15 at 23.23.31.png
    │   ├── Screenshot 2023-10-25 at 00.24.31.png
    │   ├── Screenshot 2023-11-08 at 02.04.47.png
    │   ├── Screenshot 2023-11-08 at 02.44.34.png
    │   ├── Screenshot 2023-11-08 at 02.54.51.png
    │   ├── Screenshot 2024-05-20 at 23.56.09.png
    │   ├── View recent photos.heic
    │   └── image0.heic
    └── iMessage;-;+44...946;;;SMS;-;+44...946.pdf

29 directories, 160 files
```

**Environment (please complete the following information):**
 - OS: 15.2 Beta (24C5079e)
 - Version: 2.4.10 Darwin/arm64

**Additional context**

Hope this is useful!

Appreciate you

**Comment by tagatac** [comment]
Thanks for the report! I think the issue is with the combination of the options `--pdf` and  `--copy-attachments`. I just tried that combo, and the resultant PDFs do not contain any of the images. Stay tuned for a fix. In the meantime, you can work around this by not using that specific combination of options.

**Comment by tagatac** [comment]
@ldexterldesign thanks again for the report. That was a big bug. It should be fixed in v2.4.11. Please re-open if it isn't fixed.
